aio/test-resources/plugin-crd.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: plugins.dashboard.k8s.io
+spec:
+  group: dashboard.k8s.io
+  scope: Namespaced
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  names:
+    kind: Plugin
```

Comment:
Not sure how conflicts are handled. If someone else registered a plugin, how would you then list plugins?

Should this be dashboardplugin?

Reply:
Because of the group name, I think there won't be any issues/clashes.

Reply:
A quote from leggitt on a similar issue: "shortnames can be used within different groups, like kubectl get svc.mygroup.io" So, yeah. I think your right. its a non issue. Thanks. :)